frontend/src/index.scss

```diff
@@ -5,3 +5,62 @@ body {
   margin: 0;
   padding: 0;
 }
+
+.mapboxgl-user-location-dot {
```

Comment:
This is gross and really shouldn't be here, but putting it into the right place would take a long time.

Cleaning this up should be a part of the "AdminCrossingMap - Styling Map" story.

Reply:
cool, will you add that to Asana if you haven't already and link back to this comment?